docs/_includes/generated/python_configuration.html

```diff
@@ -0,0 +1,21 @@
+<table class="table table-bordered">
```

Comment:
I think we can put it in the `Deployment & Operations/Configuration`, this is because this configuration is not only been used by Table API, but will also be used by DataStream Python UDFs, etc, in the future. It is more like an operator configurations. 
What do you think?

Reply:
Make sense, thanks a lot for the suggestions!